tests/e2e/constants/BASE_TEST_CONSTANTS.ts

```diff
@@ -130,6 +132,16 @@ export const BASE_TEST_CONSTANTS: {
 	 */
 	DELETE_WORKSPACE_ON_FAILED_TEST: process.env.DELETE_WORKSPACE_ON_FAILED_TEST === 'true',
 
+	/**
+	 * stop and remove workspace if a test is successful.
+	 */
+	DELETE_WORKSPACE_ON_SUCCESSFUL_TEST: process.env.DELETE_WORKSPACE_ON_SUCCESSFUL_TEST !== 'false',
+
+	/**
+	 * restart existing workspace.
+	 */
+	IS_RESTART_EXISTING_WORKSPACE: process.env.IS_RESTART_EXISTING_WORKSPACE === 'true',
```

Comment:
IMHO, the more relevant constant name is `OPEN_EXISTING_WORKSPACE_INSTEAD_OF_CREATION_NEW`

Reply:
Fixed

Reply:
thanks